tests/test_utils.py

```diff
@@ -136,15 +136,44 @@ async def mutate_registrations():
     assert cb4.exec_count == 1  # Registered during previous invoke(), was called
 
 
+# Timeout within 2 seconds (not)
+@pytest.mark.timeout(2)
 def test_random_port():
     assert random_port(9000, 9000) == 9000
 
+    # Starting port
+    port = 9001
+    # Test `k` continguous ports
+    k = 10
```

Comment:
Would be clearer to call this `num_ports` (when reading code below that uses `k`, I had to keep scrolling back up to see what it meant)